client/planning-extension/src/extension.ts

```diff
@@ -121,6 +121,31 @@ const extension: IExtension = {
                 entities: {
                     article: {
                         getActions: (item) => [
+                            {
```

Comment:
If conditions aren't satisfied - the action should not be displayed. The way you implemented it - it is still displayed, but clicking it doesn't do anything :smile:

Reply:
I've come to the same realisation, that's why in the fulfil assignment action I added an `else` clause to notify the user they can't execute the action. For the future I can adjust the condition easily.

Reply:
What I don't get is why don't you modify `getActions` function so that if conditions aren't satisfied - it would return an empty array and no actions would be displayed to a user as a result?